YOUR_PACKAGE/__main__.py

```diff
@@ -0,0 +1,44 @@
+"""
+Example of how to import and use the brewblox service
+"""
+
+import logging
+from typing import Type
+
+from aiohttp import web
+
+from brewblox_service import events, service
+
+routes = web.RouteTableDef()
+LOGGER = logging.getLogger(__name__)
+
+
+@routes.get('/hello')
+async def greet(request: Type[web.Request]):
+    return web.Response(body='Hello world!')
+
+
+async def on_message(queue, key, message):
```

Comment:
add docstrings to describe what on_message does (mentioning rabbitmq etc) and document arguments.
also applies to function above.

Reply:
Fixed